cobc/cobc.c

```diff
@@ -4198,6 +4204,9 @@ process_command_line (const int argc, char **argv)
 		cb_flag_trace = 1;
 		cb_flag_source_location = 1;
 	}
+	if (cb_flag_trace) {
```

Comment:
add `|| cb_flag_dump != COB_DUMP_NONE` here, then you can remove it from `cobc_def_dump_opts ()` completely